src/plugins/rename/rename.c

```diff
@@ -203,23 +228,16 @@ static Key * restoreKeyName (Key * key, const Key * parentKey, const Key * confi
 	{
 		if (strcmp (keyString (origNameKey), keyName (key)))
 		{
-			int hasSync = keyNeedSync (key); // test_bit(key->flags, KEY_FLAG_SYNC);
```

Comment:
Why did you remove these lines which restore the sync bit? They are needed for #404. We should always put comments above lines that fix a bug so that everyone know what they are for.